src/loaders/google_map_loader.js

```diff
@@ -58,13 +58,31 @@ export default (bootstrapURLKeys, heatmapLibrary) => {
       }
     }
 
+    // Support for older version using heatMapLibrary option
+    if (heatMapLibrary) {
```

Comment:
This should be `heatmapLibrary`, got an error.

Reply:
> Just found a bunch of bugs while running your code that didn't notice before, please be more careful in the future

Oh boy yeah, ok will be more diligent later.